{{cookiecutter.repo_name}}/templates/python/{{cookiecutter.repo_name}}.py

```diff
@@ -0,0 +1,49 @@
+import sys
```

Comment:
As far as I can tell, this file isn't actually being copied into the project directory.

Reply:
I'll check it out! Was this on Windows also?

Reply:
I've done `cookiecutter https://github.com/janash/MDI_Driver_Cookiecutter.git -c mdimechanic` on both Windows and Ubuntu, and I get the same result.  I only see a file called {{cookiecutter.repo_name}} with nothing in it and no extension.